### PR TITLE
Update symmetric key sample

### PR DESCRIPTION
**Pull request by schoims**

### CODE REVIEW

File: provisioning/device/samples/How To/SymmetricKeySample/Parameters.cs [hunk 1]
```diff
@@ -32,13 +32,6 @@ internal class Parameters
...
         public string PrimaryKey { get; set; }
 
-        [Option(
-            'e',
-            "EnrollmentType",
-            Default = EnrollmentType.Individual,
-            HelpText = "The type of enrollment: Individual or Group")]
-        public EnrollmentType EnrollmentType { get; set; }
-
         [Option(
```
Comment (schoims) [inline provisioning/device/samples/How To/SymmetricKeySample/Parameters.cs:35]: Remove an unused parameter

File: provisioning/device/samples/How To/SymmetricKeySample/ProvisioningDeviceClientSample.cs [hunk 2]
```diff
@@ -27,12 +27,11 @@ public async Task RunSampleAsync()
...
 
-            // For individual enrollments, the first parameter must be the registration Id, where in the enrollment
-            // the device Id is already chosen. However, for group enrollments the device Id can be requested by
-            // the device, as long as the key has been computed using that value.
-            // Also, the secondary could be included, but was left out for the simplicity of this sample.
+            // For group enrollments, the second parameter must be the derived device key.
+            // See the ComputeDerivedSymmetricKeySample for how to generate the derived key.
+            // The secondary key could be included, but was left out for the simplicity of this sample.
             using var security = new SecurityProviderSymmetricKey(
-                _parameters.Id,
```
Comment (schoims) [inline provisioning/device/samples/How To/SymmetricKeySample/ProvisioningDeviceClientSample.cs:35]: For both individual and group enrollments, the registration id must be supplied.

**Review by drwill-ms — APPROVED** [review]
Any impact to docs? Maybe Jimaco will update them for you. Otherwise, see @patilsnr or
@ngastelum-ms for insight on how to update them.

**Pull request merged**